ansible/group_vars/all.yml

```diff
@@ -96,3 +96,21 @@ dns_replicas: 1
 
 # See kube documentation for apiserver runtime config options.  Example below enables HPA, deployments features.
 # apiserver_extra_args: "--runtime-config=extensions/v1beta1/deployments=true"
+
```

Comment:
lines 99-101 already exist and should not be in this pr: https://github.com/kubernetes/contrib/blob/master/ansible/group_vars/all.yml#L40-L41